src/commands/help.js

```diff
@@ -9,27 +9,61 @@ const msgDefaults = {
   icon_emoji: config("ICON_EMOJI"),
 };
 
-let attachments = [
+/* BUTTON ACTIONS - FULL OR CUSTOM REPORT */
+let blocks = [
```

Comment:
This `block` object was copied/pasted from the [Block Kit Builder](https://app.slack.com/block-kit-builder/T9AD02VV0#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22Hey%20there%20%F0%9F%91%8B%20I'm%20Lighthouse.%20%5Cn%5CnI'm%20here%20to%20help%20you%20identify%20and%20fix%20common%20problems%20that%20affect%20your%20site's%20performance,%20accessibility,%20and%20user%20experience.%20Learn%20more%20about%20me%20%3Chttps://developers.google.com/web/tools/lighthouse/?utm_source=devtools%7Chere%3E.%5Cn%5Cn%22%7D,%22accessory%22:%7B%22type%22:%22image%22,%22image_url%22:%22https://www.leankoala.com/media/249/download/lighthouse.png?v=1%22,%22alt_text%22:%22lighthouse%20logo%22%7D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22*What%20would%20you%20like%20to%20do?*%5Cn%20You%20can%20choose%20to%20run%20the%20full%20report,%20or%20select%20the%20specific%20audits%20you'd%20like%20to%20run.%20Information%20on%20all%20the%20audits%20can%20be%20found%20%3Chttps://web.dev/learn/#lighthouse%7Chere%3E.%20%22%7D%7D,%7B%22type%22:%22actions%22,%22elements%22:%5B%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Run%20full%20report%22%7D,%22value%22:%22full_report%22%7D,%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Customize%20report%22%7D,%22value%22:%22custom_report%22%7D%5D%7D%5D%7D) that Slack provides where you can build Slack interactions. This is to be added to the payload to display the formatted text, images, and interactive buttons.

> ![image](https://user-images.githubusercontent.com/23146829/97525277-079faf00-196c-11eb-8290-7af2a52010bb.png)